bots/tests-scan

```diff
@@ -251,7 +277,7 @@ def prioritize(status, title, labels, priority, context):
 
         # Prefer "local" operating system
         elif os.environ.get("TEST_OS", "?") not in context:
-            priority -= random.randint(1, 2)
+            priority -= 1
```

Comment:
FTR, I think landing these priority computation changes now, without the queue, should be harmless enough (wrt. a possible split of this PR).